content/20.treatments.md

```diff
@@ -380,15 +380,23 @@ Post-infection, these levels return to baseline ranges [@doi:10.1177/00369330730
 
 A recent meta-analysis found consistent support for regular vitamin C supplementation reducing the duration of the common cold, but that supplementation with vitamin C (> 200 mg) failed to reduce the incidence of colds [@doi:10.1002/14651858.CD000980.pub4].
 Individual studies have found Vitamin C to reduce the susceptibility of patients to lower respiratory tract infections such as pneumonia [@doi:10.1097/00006454-199709000-00003].
-A recent meta-analysis has demonstrated in twelve trials that vitamin C supplementation reduced the length of stay of patients in intensive care units (ICUs) by 7.8% (95% CI: 4.2% to 11.2%; p = 0.00003).
+Another meta-analysis has demonstrated in twelve trials that vitamin C supplementation reduced the length of stay of patients in intensive care units (ICUs) by 7.8% (95% CI: 4.2% to 11.2%; p = 0.00003).
 Furthermore, high doses (1-3 g/day) significantly reduced the length of an ICU stay by 8.6% in six trials (p = 0.003).
 Vitamin C also shortened the duration of mechanical ventilation by 18.2% in three trials in which patients required intervention for over 24 hours (95% CI 7.7% to 27%; p = 0.001) [@doi:10.3390/nu11040708].
```

Comment:
Belated comment, but perhaps it would be good to specify more exactly what you mean by vitamin C since there are a few different ways these studies go (could be "not being Vitamin C deficient", normal dosage, high dosage)

Reply:
Hey @rando2 The reason I haven't been specific is that there is difficulty in conducting meta-analyses of supplements as various doses are used, some studies account for deficiency, some don't. There are a lot of issues, which the meta-analyses themselves comment on too. 

Do you think a sentence on that is warranted, or just provide more detail?

Thanks!

Reply:
Oh, if it's intentional, then that's totally fine with me! It seems like such an interesting body of literature, there is a lot going on!

Reply:
Its continuing to grow, I have to add covid trials to the zinc and vit D section haha